tldw_Server_API/app/core/Chat/chat_orchestrator.py

```
@@ -531,6 +532,36 @@ def chat(
+        injected_command_system_text: Optional[str] = None
+        original_message = message
+        if command_router.commands_enabled() and isinstance(message, str):
+            parsed = command_router.parse_slash_command(message)
+            if parsed:
+                cmd_name, cmd_args = parsed
+                # Build minimal context; user_identifier may be None
+                auth_user_int = None
+                try:
+                    if llm_user_identifier is not None:
+                        auth_user_int = int(llm_user_identifier)  # best-effort parse for RBAC
+                except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)